base/applications/mscutils/servman/lang/es-ES.rc

```diff
@@ -334,6 +322,7 @@ END
 STRINGTABLE
 BEGIN
     IDS_APPNAME "Administrador de servicios de ReactOS"
+    IDS_APPAUTHORS "Copyright (C) 2005-2007 by Ged Murphy (gedmurphy@reactos.org)"
```

Comment:
```suggestion
    IDS_APPAUTHORS "Copyright (C) 2005-2007 creado por Ged Murphy (gedmurphy@reactos.org)"
```

Reply:
ok, even if i'd prefer to have translation updated aside from this PR.